actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+*   Raise ArgumentError on Parameter.new
```

Comment:
Raise \`ArgumentError\` when passing anything that doesn't respond to \`with_indifferent_access\` to \`ActionController::Parameter.new\`.